### PR TITLE
Release notes XTC 72

### PR DESCRIPTION
**Pull request by jowerner**

### CODE REVIEW

File: content/en/xtc/xtc-release-notes/v72.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Platform
+
+### New Feature: Project Documentation
+
+Projects now provide a documentation page. See the new entry *Documentation* in the navigation bar. Use this page to share general information about the project with other project members, for example, testing goals and success criteria, test windows, contact information, news, etc. The content of the page can be formatted nicely using Markdown.
```
Comment (h-arlt) [inline content/en/xtc/xtc-release-notes/v72.md:21]:
```suggestion
Projects now provide a documentation page. See the new entry *Documentation* in the side navigation. Use this page to share general information about the project with other project members, for example, testing goals and success criteria, test windows, contact information, news, etc. The content of the page can be nicely formatted using Markdown.
```

File: content/en/xtc/xtc-release-notes/v72.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Members with the role of *Reviewer* (and higher) can view the documentation page. In order to edit the page, you need to be *Test Manager* or *Project Administrator*.
```
Comment (h-arlt) [inline content/en/xtc/xtc-release-notes/v72.md:23]:
```suggestion
Members having the project role *Reviewer* (and higher) can view the documentation page. In order to edit the page, you need to have project role *Test Manager* or *Project Administrator*.
```

**Review by h-arlt — APPROVED** [review]
Just two suggestions; not required for merge.

File: content/en/xtc/xtc-release-notes/v72.md (hunk 2 above)
Comment (jowerner) [reply content/en/xtc/xtc-release-notes/v72.md:23]: Online grammar/style checkers replaced "having" with "with", so I suggest to trust them with this.

**Review by h-arlt — APPROVED** [review]

**Pull request merged**